recipe/meta.yaml

```diff
@@ -1,63 +1,90 @@
-{% set name = "onnx" %}
-{% set version = "1.14.1" %}
+{% set version = "1.15.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: onnx-split
   version: {{ version }}
 
 source:
   url: https://github.com/onnx/onnx/archive/v{{ version }}.tar.gz
-  sha256: e296f8867951fa6e71417a18f2e550a730550f8829bd35e947b4df5e3e777aa1
+  sha256: c757132e018dd0dd171499ef74fca88b74c5430a20781ec53da19eb7f937ef68
   patches:
+    - undo_definitions_of_cxx_standard.patch
+    - cinttypes_cpp2py_export.patch
     - 0001-Link-to-abseil_dll.patch  # [win and (libprotobuf != "3.21")]
 
 build:
-  number: 4
-  entry_points:
-    - check-model = onnx.bin.checker:check_model
-    - check-node = onnx.bin.checker:check_node
-    - backend-test-tools = onnx.backend.test.cmd_tools:main
+  number: 0
 
-requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
-    - pybind11                               # [build_platform != target_platform]
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - cmake
-    - make
-    - libprotobuf
-    - sed  # [unix]
-    - m2-sed  # [win]
-  host:
-    - python
-    - pip
-    - protobuf
-    - libprotobuf
-    - pytest-runner
-    - ninja
-    - pybind11
-    - numpy
-  run:
-    - python
-    - protobuf
-    - {{ pin_compatible('numpy') }}
-    - typing-extensions >=3.6.2.1
+outputs:
+  - name: libonnx
+    script: build_libonnx.sh  # [unix]
+    script: bld_libonnx.bat   # [win]
+    build:
+      run_exports:
+        - {{ pin_subpackage('libonnx', max_pin='x.x') }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - libprotobuf
+        - cmake
+        - make  # [unix]
+      host:
+        - libprotobuf
+        - libabseil  # [win]
```

Comment:
I guess this should be present also on non-Windows platforms, so the correctly run_exports are exported for abseil (as it is a public dependency of libprotobuf).

Reply:
the linking analysis at the end doesn't make libaseil appear on unix

Reply:
i think this is due to the fact that linux uses shared libraries

Reply:
For reference, this was the discussion on libprotobuf now requiring linking to libabseil: https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/4075#issuecomment-1562294904 .